logging/google/cloud/logging/_helpers.py

```diff
@@ -46,3 +52,30 @@ def entry_from_resource(resource, client, loggers):
         return ProtobufEntry.from_api_repr(resource, client, loggers)
 
     raise ValueError('Cannot parse log entry resource.')
+
+
+def retrieve_metadata_server(metadata_key):
+    """Retrieve the metadata key in the metadata server.
+
+    See: https://cloud.google.com/compute/docs/storing-retrieving-metadata
+
+    :type metadata_key: str
+    :param metadata_key: Key of the metadata which will form the url.
```

Comment:
Yes the metadata key can include the query parameters, I just can't think of a name to represent the whole thing. And in most circumstances the metadata key will just be the metadata key. I added documentation about this parameter.